README_fr.md

```diff
@@ -0,0 +1,1641 @@
+
+Par exemple, si vous voulez **laisser le commit le plus vieux (le premier) tel quel et combiner tous les commits suivants avec le deuxième plus vieux**, vous devez remplacer les lettres à côté de chaque commit sauf le premier et le deuxième par un `f` :
+
+```vim
+pick a9c8a1d Un peu de refactor
+pick 01b2fd8 Nouvelle feature géniale
+f b729ad5 fixup
+f e3851e8 autre fix
+```
+
+Si vous voulez combiner ces commits **et renommer le commit**, il vous faudra ajouter un `r` à côté du deuxième commit ou utiliser simplement `s` au lieu de `f` :
+
+```vim
+pick a9c8a1d Un peu de refactor
+pick 01b2fd8 Nouvelle feature géniale
+s b729ad5 fixup
+s e3851e8 autre fix
+```
+
+Vous pouvez ensuite renommer le commit dans le prochain éditeur de texte qui s'affiche :
+
+```vim
+Nouvelles fonctionnalités encore plus géniales
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+# rebase in progress; onto 8074d12
+# You are currently editing a commit while rebasing branch 'master' on '8074d12'.
+#
+# Changes to be committed:
+#   modified:   README.md
+#
+
+```
+
+Si tout se passe bien, vous devriez voir quelque chose comme ceci :
+
+```sh
+(master)$ Successfully rebased and updated refs/heads/master.
+```
+
+#### Stratégie de fusion sûre
+`--no-commit` fait une fusion mais prétend que la fusion a échoué et ne commit par automatiquement, laissant la chance à l'utilisateur d'inspecter plus et de bidouiller plus les résultats de la fusion avant de commiter. `no-ff` garde une preuve qu'une branche de fonctionnalité a auparavant existé, gardant ainsi l'historique du projet consistant :
+
+```sh
+(master)$ git merge --no-ff --no-commit ma-branche
+```
+
+#### J'ai besoin de fusionner deux branches en un seul commit
+
+```sh
+(master)$ git merge --squash ma-branche
+```
+
+<a name="rebase-unpushed-commits"></a>
+#### Je veux combiner les commits non poussés uniquement
+
+Parfois, vous avez plusieurs commits en cours que vous souhaitez combiner avant de les pousser sur l'upstream. Vous ne voulez pas combiner des commits qui ont déjà été poussés sur l'upstream par accident, car quelqu'un d'autre a peut-être déjà effectué des commits qui y font référence :
+
+```sh
+(master)$ git rebase -i @{u}
+```
+
+Cela fera un rebase interactif que liste seulement les commits que vous n'avez pas poussé, afin que cela soit plus sûr de réordonner/corriger/combiner n'importe lesquels de la liste.
+
+#### J'ai besoin d'annuler la fusion
+
+Parfois, une fusion peux produire des problèmes pour certains fichiers. Dans ces cas là, nous pouvons utiliser l'option {code0}abort{/code0} afin d'abonner le processus de résolution de conflits en cours, afin d'essayer de reconstruire l'état qu'on avait avant la fusion :
```

Comment:
```suggestion
Parfois, une fusion peux produire des problèmes pour certains fichiers. Dans ces cas là, nous pouvons utiliser l'option `abort` afin d'abonner le processus de résolution de conflits en cours, afin d'essayer de reconstruire l'état qu'on avait avant la fusion :
```
Just another odd code format tag.
Maybe I'm wrong and they're actually a thing?

Reply:
Thanks! Actually these are leftovers from GitLocalize, which uses different tags for formating. I missed some when converting them by hand